packages/nuxt-locokit/src/runtime/pages/workspace/index.vue

```diff
@@ -117,13 +127,26 @@ await workspacesStore.updateCurrentWorkspace({
 if (typeof workspaces.value === 'undefined') {
   await workspacesStore.updateWorkspaces()
 }
+definePageMeta({
+  redirect: {
+    name: ROUTES_NAMES.WORKSPACE.DASHBOARD,
+  },
+})
 
 const closeWorkspaces = () => {
   showWorkspaces.value = false
 }
 
 onMounted(() => {
   document.addEventListener('click', closeWorkspaces)
```

Comment:
Mmm, I don't understand what's the purpose of this ?
Could you remind me ?
Why didn't we do it by using vuejs directive `v-on` on the related element ? We add it to the `document` directly, is this really what we want to do ?

Reply:
yes